src/base/init/init.c

```diff
@@ -351,7 +351,7 @@ void low_mem_init(void)
   if (config.cpu_vm == CPUVM_KVM || config.cpu_vm_dpmi == CPUVM_KVM) {
     init_kvm_monitor(memsize + dpmi_size);
     mmap_kvm(MAPPING_INIT_LOWRAM, mem_base, memsize + dpmi_size,
-	    PROT_READ | PROT_WRITE, 0);
+	    PROT_NONE, 0);
```

Comment:
Oh, maybe putting that (and only that,
not the `init_kvm_monitor()` line above)
into mem_reserve() so that it uses the
`prot` variable to make sure the prot is
always correct?
Then init_kvm_monitor() could probably
go elsewhere, to cpu_init or whatever
else.

Reply:
Done now. Yes init_kvm_monitor() can be called earlier, just needs to use a fixed monitor mapping address (very high up, at 0xff000000).

I made sure to do:
`test/test_dos.py --require-attr=cputest PPDOSGITTestCase`
to avoid silly regressions this time!

Reply:
ok, squashed that commit